SPEC.md

```diff
@@ -764,9 +764,9 @@ Layout of the CRC tag:
 ```
         tag                                    data
 [--      32      --][--      32      --|---        variable length        ---]
-[1| 3| 8 | 10 | 10 ][--      32      --|---            (size)             ---]
+[1| 3| 8 | 10 | 10 ][--      32      --|---          (size - 4)           ---]
```

Comment:
Should this actually be `(size - 32)` since the other fields are in bits?

Reply:
yes, this field should in bits, but size is in bytes, maybe `(size * 8 - 32)` is better ?

Reply:
Ah! Good point, you're right that is better.